lerna.json

```diff
@@ -1,5 +1,6 @@
 {
   "npmClient": "yarn",
+  "npmClientArgs": ["--ignore-engines"],
```

Comment:
Don't know how I feel about this. Should we just drop support for Node 6/8 instead?

Reply:
I was gonna bring this up for discussion tomorrow but I am fine either way. What's the impact of dropping Node 6/8? If we go that route, should we drop support for all the packages in this repo or just the affected ones in this PR?

Reply:
Looks like npm considers the ["engines" flag](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) as advisory and will only generate warnings in this scenario, but yarn gives out errors. I would support dropping Node 6/8 if we feel this change is a risk which our tests cannot catch when testing with Node 6/8. Or either there is no significant impact on dropping support for Node 6/8. Thoughts?

Reply:
We've seen issues in the past with other Node libs where we had to use older implementations of core functionality because we supported older Node versions. Leads to maintenance and TODOs. Prefer to drop Node 6/8 everywhere as this meets the criteria where we're doing workarounds to support older versions. Will check with product.